app/src/main/java/io/intrepid/webservicesexercise/view/View.java

```diff
@@ -0,0 +1,11 @@
+package io.intrepid.webservicesexercise.view;
+
+/**
+ * Created by abassbayo-awoyemi on 1/11/17.
+ */
+
+public interface View {
```

Comment:
when you have an interface for view and presenter, it makes sense to define them together in a separate Contract class